.github/workflows/release-message-release.yml

```diff
@@ -0,0 +1,45 @@
+name: "[Release] message release"
+
+permissions:
+  id-token: write # for fetching the OIDC token
+  contents: read # for actions/checkout
+
+on:
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  release-message-release:
+    if: ${{ github.repository == 'trezor/trezor-suite' && (github.ref == 'refs/heads/release-message-release-production' || github.ref == 'refs/heads/release-message-release-develop') }}
+    environment: ${{ github.ref == 'refs/heads/release-message-release-production' && 'production-message-release' || 'develop-message-release' }}
+    runs-on: ubuntu-latest
+    env:
+      RELEASE_ENV: ${{ github.ref == 'refs/heads/release-message-release-production' && 'production' || 'develop' }}
+      AWS_REGION: "eu-central-1"
+      AWS_CLOUDFRONT_ID: E1ERY5K2OTKKI1
+      ROLE_TO_ASSUME: ${{ github.ref == 'refs/heads/release-message-release-production' && 'arn:aws:iam::538326561891:role/gh_actions_suite_production_message_release' || 'arn:aws:iam::538326561891:role/gh_actions_suite_develop_message_release' }}
```

Comment:
@vdovhanych I assume data in this part and probably other parts of the file will have to be different.
For now just putting this as a placeholder.